server/mcp_server.py

```
@@ -171,129 +235,79 @@ async def _api(method: str, path: str, *, params: dict | None = None,
-@mcp.resource("aurora://health")
-async def health() -> dict:
-    """Aurora system health: database, Redis, Weaviate, Celery status."""
-    return await _api("GET", "/health/")
+_GATED_NAMES = {spec.name: spec for spec in TIER2_TOOLS}
+_original_list_tools = mcp.list_tools  # bound method
 
 
-# ---------------------------------------------------------------------------
-# Prompts
-# ---------------------------------------------------------------------------
+async def _filtered_list_tools():  # type: ignore[no-redef]
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_filtered_list_tools`

(ANN202)